packages/@ama-sdk/create/README.md

```diff
@@ -0,0 +1,24 @@
+<h1 align="center">Otter extractor</h1>
+<p align="center">
+  <img src="https://raw.githubusercontent.com/AmadeusITGroup/otter/main/.attachments/otter.png" alt="Super cute Otter!" width="40%"/>
+</p>
+
+## Create an SDK package
+
+This package is simplifying the start of new SDK repository.
+
+## Usage
+
+### NPM
+
+```shell
+npm create @ama-sdk <package-name> -- [--specPath=./path/to/spec.yaml]
+```
+
+### Yarn
+
+```shell
+yarn create @ama-sdk <project-name> [--specPath=./path/to/spec.yaml]
```

Comment:
in the sdk generator schematics it is --spec-path, why the change?

Reply:
Wrong copy past in the doc, the parameter is actually `--spec-path`